src/style.css

```diff
@@ -1,3 +1,125 @@
 body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  --button-color: rgba(0, 172, 220, 1);
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/45b2a61a-30ef-499e-9d55-f3699b890cd1)
rgba is redundant here - use rgb instead